### ISSUE TITLE
Errors at starting Vim

### ISSUE THREAD

**Issue by mariomenjr** [erkrnt/awesome-streamerrc#9]
After watching your [video](https://www.youtube.com/watch?v=n9k9scbTuvQ) on YouTube, I copied the [init.vim](https://github.com/erkrnt/awesome-streamerrc/blob/master/ThePrimeagen/init.vim) file as my ~/.vimrc and then:

- Exec `rm -rf ~/.vim`
- Exec `mkdir ~/.vim/undodir -p`
- Installed Plug from https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
- Run `:PlugInstall`

After all that, I could see how the Vim appearance and functionality improved. However, I am always seeing these errors when Vim is open:

![image](https://user-images.githubusercontent.com/1946936/89715339-e8d7e080-d959-11ea-995d-46a1a1f200f1.png)

I googled for some time, but could not find anything conclusive. Do you have an idea what may I be missing?

Notes:

- Vim 8.2.1185
- Manjaro 20.0.3

**Comment by h-khoza** [comment]
![Screenshot from 2020-08-09 09-10-58](https://user-images.githubusercontent.com/51496873/89726952-d07ec900-da20-11ea-9227-b2809bdcb296.png)
I am also facing the same issue

**Comment by h-khoza** [comment]
I manage to get rid of the errors by removing the "nvim-apm" plugin, but now it doesn't highlight when I yank text, also "set kscb" on line 2 it says invalid syntax.

**Comment by erkrnt** [comment]
I believe this is related to @ThePrimeagen's vimrc - he might need to take a look.

**Comment by ThePrimeagen** [comment]
Yes, I can take a look.  @hendry3k , the first thing you should do is _remove all references to vim.apm_ anywhere.  This should _fix_ the problem.  The problem is that I may have some config that you dont due to apm.

But I can look tonight :)
